randovania/games/samus_returns/logic_database/Area 4 Crystal Mines.txt

```diff
@@ -625,7 +625,7 @@ Extra - asset_id: collision_camera_009
   * Tunnel to Gamma Arena/Tunnel to Gawron Groove
   > Tunnel to Basalt Basin (Right)
       All of the following:
-          Morph Ball and Super Missile and Varia Suit
+          Super Missile and Varia Suit
```

Comment:
why was morph ball removed here? it doesn't change logic, but is clearer IMO.

Reply:
I just see it as being redundant. Preference and consistent in other places.

Reply:
feels too small for me to meaningfully argue about it lol